src/modules/Tasks.js

```diff
@@ -0,0 +1,113 @@
+  };
+
+  //
+  const formValidation = () => {
+    // eslint-disable-next-line no-unused-expressions
+    (textInput.value === '') ? alert('Please enter a task') : addTodo();
+  };
+
+  const removeTodo = (index) => {
+    todoArray.splice(index, 1);
+    // eslint-disable-next-line no-use-before-define
+    renderTodo();
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const editTodo = (index, newDescription) => {
+    todoArray[index].description = newDescription;
+    // eslint-disable-next-line no-use-before-define
+    renderTodo();
+  };
+
+  // eslint-disable-next-line no-use-before-define
+  let renderTodo = () => {
+    const todoListItems = document.querySelector('.todo-list-items');
+    todoListItems.innerHTML = '';
+
+    todoArray.forEach((todo) => {
+      const todoItem = document.createElement('div');
+      todoItem.classList.add('todo-item');
+      todoItem.innerHTML = `
+                              <div class="todo">
+                              <input id="checkbox" type="checkbox" class="checkbox" ${todo.completed} ? 'checked' : ''}>
+                              <span class="todo-description" type="submit" contenteditable="true">${todo.description}</span>
+                              <div class="dots"></div>
+                              </div>
+                              <hr>
+                              `;
+
+      todoListItems.appendChild(todoItem);
+
+      const dot = todoItem.querySelector('.dots');
+      dot.addEventListener('click', (e) => {
+        e.preventDefault();
+        const index = todoArray.indexOf(e.target.parentNode.parentNode.parentNode);
+        removeTodo(index);
+        localStorage.setItem('todoArray', JSON.stringify(todoArray));
+      });
+
+      const checkbox = todoItem.querySelector('.checkbox');
+      // eslint-disable-next-line no-unused-vars
+      checkbox.addEventListener('click', (e) => {
+        todo.completed = true;
+        todoArray = _.filter(todoArray, (o) => !o.completed);
+        localStorage.setItem('todoArray', JSON.stringify(todoArray));
+      });
+
+      const clearBtn = document.querySelector('.btn');
+
+      clearBtn.addEventListener('click', (e) => {
+        e.preventDefault();
+        renderTodo();
+      });
+    });
+  };
+};
+
+export default Tasks;
+
+export const getTodoArray = () => {
+  const todoArray = JSON.parse(localStorage.getItem('todoArray'));
+  const todoListItems = document.querySelector('.todo-list-items');
+  todoArray.forEach((todo) => {
+    const todoItem = document.createElement('div');
+    todoItem.classList.add('todo-item');
+    todoItem.innerHTML = `
+                            <div class="todo">
+                            <input id="checkbox" type="checkbox" class="checkbox" ${todo.completed} ? 'checked' : ''}>
+                            <span class="todo-description" type="submit" contenteditable="true">${todo.description}</span>
+                            <div class="dots"></div>
```

Comment:
:point_right: Please add the hover effect on all icons as shown in the given style 
![issue0](https://user-images.githubusercontent.com/54817285/186579036-4d89fed6-6438-4c49-8905-e4eae1f608c2.png)